src/app/admin/clients/page.tsx

```
@@ -221,81 +221,91 @@ export default function AdminClientsPage() {
+                        break
+                    case 'INDIVIDUAL':
+                        if (client.clientType !== 'INDIVIDUAL') return false
+                        break
+                    case 'ACTIVE':
+                        if (client.status !== 'ACTIVE') return false
+                        break
+                    case 'INACTIVE':
+                        if (client.status !== 'INACTIVE') return false
+                        break
+                    case 'RECENT':
+                        const lastActive = new Date(client.lastActive)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)